### PR TITLE
Fix broken mempool_setup_chainstate test

### PR DESCRIPTION
**Pull request by jferrant**
Not sure how this PR passed CI: https://github.com/stacks-network/stacks-core/pull/4149 because when I investigated closer...it caused a failed test (mempool_setup_chainstate). This PR fixes the failing test.

### CODE REVIEW

File: testnet/stacks-node/src/tests/mempool.rs [hunk 1]
```diff
@@ -843,15 +827,17 @@ fn mempool_setup_chainstate() {
...
                     .will_admit_mempool_tx(
                         &NULL_BURN_STATE_DB,
                         consensus_hash,
                         block_hash,
                         &tx,
                         tx_bytes.len() as u64,
                     )
-                    .unwrap();
+                    .unwrap_err();
+                eprintln!("Err: {:?}", e);
```
Comment (xoloki) [inline testnet/stacks-node/src/tests/mempool.rs:839]: This prinkln can go away

**Review by xoloki — APPROVED** [review]
Looks reasonable modulo the extraneous eprintln, but the test is still falling

File: testnet/stacks-node/src/tests/mempool.rs (hunk 1 above)
Comment (jferrant) [reply testnet/stacks-node/src/tests/mempool.rs:839]: Its there for consistency with the rest of the test.

**Comment by jferrant** [comment]
> Looks reasonable modulo the extraneous eprintln, but the test is still falling

I think this is a caching problem. Local runs show it passes no problem. And the error it returns is not making sense. Hopefully retriggering will fix it.

**Review by jcnelson — APPROVED** [review]

**Review by saralab — APPROVED** [review]

**Comment by jcnelson** [comment]
My PR ate this PR

**Pull request closed without merging**